spec/javascripts/components/step-by-step-nav-spec.js

```diff
@@ -448,7 +448,7 @@ describe('A stepnav module', function () {
 
     it("sets the show/hide link text to 'hide'", function () {
       var $step1 = $element.find('#topic-step-one');
-      expect($step1.find('.js-toggle-link')).toHaveText("Hide");
+      expect($step1.find('.js-toggle-link')).toHaveText("(Hide)");
```

Comment:
do we need another test to check for "(Show)"

Reply:
This test is to check that the text for an open step is set correctly. There should be other tests unrelated to this change that check that the text is correct when opening and closing steps.

Turns out those tests don't exist, so I've added a card to get them implemented, but that's outside the scope of this PR (https://trello.com/c/2MXu4wYS/525-extend-tests-for-show-hide-text)